tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -1064,6 +1127,7 @@ def test_positive_ping(self):
             u'Not all services seem to be up and running!'
         )
 
+    @pytest.mark.skip_if_open("BZ:1787147")
```

Comment:
Could you only skip parts of test, instead of entire thing?
If I understand correctly, only parts related to manifest and depending on manifest are blocked, everything else should work. By blocking entire thing we lose information until this bug is fixed, which might result in us noticing another regression later than we could.

Reply:
The test is the end-to-end test and currently, this issue is breaking that link of end-to-end. All above the failing assert is already being exercised in CRUD/integration tests and I believe it is not required to test them here.

So skipping the entire test is better here. What do u say ?

Reply:
These exact scenarios, when action is run as another user with admin permissions, are generally not covered by specific CRUD tests. Most of the time we work as default admin, sometimes we create new user with very specific permissions.

That was only the suggestion, I'm fine with current approach as well.